content/sensu-go/5.12/reference/rbac.md

```diff
@@ -48,6 +48,8 @@ You can use [sensuctl][2] to view all namespaces within Sensu:
 sensuctl namespace list
 {{< /highlight >}}
 
+_Note: If logged in with a non Admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```

Comment:
```suggestion
_Note: If logged in with a non-admin user, `sensuctl namespace list` will list only the available namespaces that the user has access to._
```

I see other areas of the docs site use this format. Or we can get rid of it entirely and say something like "If logged with with a user that does not have administrative access, ..."